freud/cluster.pyx

```diff
@@ -246,5 +261,19 @@ cdef class ClusterProperties(_Compute):
             &self.thisptr.getClusterSizes(),
             freud.util.arr_type_t.UNSIGNED_INT)
 
+    @_Compute._computed_property
+    def cluster_masses(self):
+        """(:math:`N_{clusters}`) :class:`numpy.ndarray`: The cluster masses."""
+        return freud.util.make_managed_numpy_array(
+            &self.thisptr.getClusterMasses(),
+            freud.util.arr_type_t.FLOAT)
+
+    @_Compute._computed_property
+    def radii_of_gyration(self):
```

Comment:
We should add explicit formulas for all properties, perhaps in the class docstring. It is reasonable to expect that the radius of gyration is computed from the gyration tensor, but our references in #778 indicate that it should be computed from the moment of inertia tensor.